libraries/chain/hotstuff/block_construction_data_flow.md

```diff
@@ -72,7 +72,7 @@ The new storage for IF is:
 struct block_header_state_core {
```

Comment:
actually there is no `block_header_state_core` anymore :-)

Reply:
I'm not sure what we gain by maintaining this MD doc when it is just a mirror of the code. Should I remove it?

Reply:
I think this .md (written by Areg btw) may still have some information (describing which post-if members are equivalent to the pre-if ones) which is not in the code, but it is true that I have not needed to look at it in some time. I agree that there is not need to keep it up-to-date, but maybe we should keep it a little longer and remove it before the release. Before removing it, I'd like to go through it and make sure that insightful info from it is replicated in code comments.

Reply:
Sounds good to me.